genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -99,6 +99,13 @@ func ResourceTfExport() *schema.Resource {
 				Default:     false,
 				ForceNew:    true,
 			},
+			"export_as_hcl": {
+				Description: "Export the config as HCL. Deprecated use export_format instead ",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
```

Comment:
Maybe remove the default in deference to the default set by `export_format`. We only really care if this attribute is actually set.